contents/ruby/Rails.md

```diff
@@ -0,0 +1,154 @@
+
+However, Rails might not be best suited for all use cases, and there are certain areas where other frameworks might be more suitable. For instance:-
+
+### Drawbacks
+
+#### Lack of Flexibility
+
+As demonstrated above, Rails follows very strict conventions, that make life much easier for developers - or so Rails thinks. It might be true that there are some developers who find Rails' convention very stifling, and, as a result, prefer to use frameworks that allow them more flexibility. Rails will still allow you to do as you please, but since the effort of undo-ing the conventions makes the process longer and less enjoyable, it might be better to avoid Rails.
+
+#### Performance Time
+
+While newer releases have combatted this problem to a large degree, many of the old releases of Rails have had a reputation for being very bloated and slow to respond. Applications made using NodeJS, or purely client-side applications with a minimal backend, tend to have a much lower delay between an action and a response. While it should be noted that the vast majority of delays in applications occur due to poor optimization, and not the inherent speed of the framework, Rails still remains more bulky and slower than many of its competitors. 
+
+#### Learning Curve
+
+Rails' tendency to do things "automagically" can also be very confusing to new users, who might be more used to configuring routes and directory structures themselves. Furthermore, it becomes much harder to go from using Rails to a different framework, as the lack of conventions in other frameworks means you will have to learn some new topics to be proficient. You also have to learn the Ruby language, and it might be harder considering that most front-end applications are in JavaScript and its variants, and you will have to juggle multiple languages while you develop your application. 
+
+## Getting Started
+
+To install Rails on your computer, you will first need to install [Ruby](https://www.ruby-lang.org/en/). After this, you simply need to run:
+
+```rb
+gem install rails
+```
+gem is a package manager for Ruby that allows you to install Rails (as well as all the other Gems in Ruby). Once you're done with the setup, you can start developing applications right away. Some resources you might find helpful are:-
+
+1. The [Ruby Guides](https://www.rubyguides.com/) is a must read, as you cannot use Rails without knowing the underlying language.
+2. The [Rails Guides](https://guides.rubyonrails.org/) are the definitive guide for Rails and you can be assured that it will always be up to date with the latest releases.
```

Comment:
> The [Rails Guides](https://guides.rubyonrails.org/) are the definitive guide for Rails and you can be assured that it will always be up to date with the latest releases.

Similar issues with singular vs plural terms here.

Reply:
I think both are fine, depending on whether you're referring to the collection of guides as a singular object (Like 'that group _is_ there') or whether it's referring to just a bunch of guides, not a collective. I'll standardize it to plural though :)